### PR TITLE
Small corrections in  Brazilian Portuguese translation and conversion…

### PR DESCRIPTION
**Pull request by edsoncunha**
… to gender-neutral language

### CODE REVIEW

**Comment by MattiSG** [comment]
Thank you very much @edsoncunha! This changeset has to be proofread by another native Brazilian Portuguese speaker before it can be merged 😃

If you know any on GitHub, please mention them here! I know @Cisneiros and @s3rgiosan already contributed, and might be able to “approve” this PR.

Also, there are a few pending contributions in #91. If you were to integrate these changes in this PR, it would be great and would avoid conflicts 😃

Please let me know if anything isn't clear and I can be of any help 😉 

Cheers!
Matti

**Comment by edsoncunha** [comment]
Hi, @MattiSG!

It seems @abdala is not interested in merging #91 anymore...

**Comment by MattiSG** [comment]
Yes, absolutely! Which is why I think it could be a good thing for you to integrate in this PR the improvements that were suggested in #91 — if you think that they are indeed improvements! :wink:

**Comment by Cisneiros** [comment]
Hi @edsoncunha. While I can vouch for the correctness of your changes, I do believe the term "pessoa(s)" has been used quite a lot. While is a very good term for making gender neutral phrasing in Portuguese, we can try to get creative in some instances where we can use a different term just so it is not so repetitive. For instance, this one could be used in one of the changes to remove the subject:

"Violações a estas regras poderão causar punição ou expulsão da conferência, sem restituição, a critério da equipe organizadora."

Maybe also use "indivíduo" a few times? And "pessoas patrocinadoras" sounds a bit odd, because usually companies sponsor, not people. Maybe something like "representantes de patrocinadores" or just "patrocinadores e patrocinadoras". What do you think?

**Comment by edsoncunha** [comment]
Thanks for your suggestions, @Cisneiros!

I agree about the repetition of "pessoas"... It's very hard to achieve neutral plural in pt-br.
"representantes de patrocinadores" sounds great too.

I only disagree about the word "indivíduos". It's tricky, because it kind of falls (not directly) into the same problem of generalizing plurals in male form.

File: index-pt-br.html [hunk 1]
```diff
@@ -98,28 +98,28 @@
 </p>
 
 <p>
-  Pessoas que patrocinam também estão sujeitas às políticas antiassédio. Em particular, elas 
+  Representantes de empresas patrocinadoras também se aplicam às políticas antiassédio. Em particular,
```
Comment (Cisneiros) [inline index-pt-br.html]: I think this would be "A representantes de empresas patrocinadoras,  também se aplicam as políticas antiassédio.

Comment (edsoncunha) [reply index-pt-br.html]: Agreed.

Comment (edsoncunha) [reply index-pt-br.html]: (but without the comma :)  )

**Comment by Cisneiros** [comment]
Ready to merge, for me!

**Comment by MattiSG** [comment]
@edsoncunha @Cisneiros thank you very much to both of you! 👏 😄

@Cisneiros next time, if you can use again GitHub's PR review interface to explicitly “Approve” the changes, it makes it slightly easier for maintainers to follow what's going on 🙂 this is really just an improvement suggestion, thanks again so much for your reactivity and review!

**Comment by MattiSG** [comment]
If any of you feels like carrying #91 over, please feel free to do so! For example by opening a new PR that takes the improvements you agree with. Otherwise, I will probably have to close it since there are now conflicts with the current version.

**Pull request merged**